analyzers/tests/SonarAnalyzer.UnitTest/Rules/DisposableNotDisposedTest.cs

```diff
@@ -36,6 +36,12 @@ public class DisposableNotDisposedTest
 
 #if NET
 
+        [TestMethod]
+        public void DisposableNotDisposed_CSharp8() =>
+            builder.AddPaths("DisposableNotDisposed.CSharp8.cs")
+                .WithOptions(ParseOptionsHelper.FromCSharp8)
+                .Verify();
```

Comment:
Can you also add a test with

await using var ignored1 = objOne.ConfigureAwait(false);


For completion?

Reply:
Added it to the C# 8 test cases.